packages/common/src/TransactionUtils.ts

```diff
@@ -76,6 +86,16 @@ export const runTransaction = async ({
   // This method does not play nicely in tests. Leave the nonce null to auto fill.
   else if (argv.network != "test") transactionConfig.nonce = await web3.eth.getTransactionCount(transactionConfig.from);
 
+  // Store the transaction nonce in the web3 object so that it can be used in the future. This enables us to fire a
+  // bunch of transactions off without needing to wait for them to be included in the mempool by manually incrementing.
+  if ((web3 as AugmentedWeb3).nonces?.[transactionConfig.from])
+    transactionConfig.nonce = ++(web3 as AugmentedWeb3).nonces[transactionConfig.from];
```

Comment:
Instead of casting in-line, could we cast at the top of the method and just use AudmentedWeb3 everywhere in this method?

Reply:
yup. done ptal